adapter/src/main/java/ar/edu/undec/adapter/data/curso/mapper/CursoMapper.java

```diff
@@ -0,0 +1,13 @@
+package ar.edu.undec.adapter.data.curso.mapper;
+
+import ar.edu.undec.adapter.data.curso.model.CursoEntidad;
+import curso.modelo.Curso;
+
+public class CursoMapper {
+    public static Curso dataCoreMapper(CursoEntidad cursoEntidad){
+        return Curso.factory(cursoEntidad.getId(), cursoEntidad.getNombre(), cursoEntidad.getFechaCierreInscripcion(),cursoEntidad.getNivel());
+    }
+    public static CursoEntidad coreDataMapper(Curso curso){
+        return new CursoEntidad(null, curso.getNombre(), curso.getFechaCierreInscripcion(), curso.getNivel());
```

Comment:
El curso siempre tendría un ID, no enviarle nul
```suggestion
        return new CursoEntidad(curso.getID(), curso.getNombre(), curso.getFechaCierreInscripcion(), curso.getNivel());
```